athena-federation-sdk/src/test/java/com/amazonaws/athena/connector/lambda/serde/ObjectMapperUtil.java

```diff
@@ -40,9 +40,12 @@ private ObjectMapperUtil() {}
 
     public static <T> void assertSerialization(Object object)
     {
+        //We expect consuming engines to manage backwards/forwards serialization. A consuming engine must use whichever
+        //SerDe is best supported. For example, if an engine is using the V2 SerDe and the connector supports the V3 SerDe,
+        //then the engine will use the V2 SerDe path for both.
         assertBaseSerialization(object);
-        assertSerializationForwardsCompatibleV2toV3(object);
-        assertSerializationBackwardsCompatibleV3toV2(object);
+        //assertSerializationForwardsCompatibleV2toV3(object);
+        //assertSerializationBackwardsCompatibleV3toV2(object);
```

Comment:
Are these intentionally commented out?

If so why do the corresponding commented out methods have new changes below?

Reply:
need to delete this and the associated blocks. We moved the logic to exist in Composite handler